cidc_api/models/models.py

```diff
@@ -561,6 +562,8 @@ class UploadJobs(CommonColumns):
     _status = Column(
         "status", Enum(*UPLOAD_STATUSES, name="upload_job_status"), nullable=False
     )
+    # A long, random identifier for this upload job
+    token = Column(String, default=uuid.uuid4)
```

Comment:
will this propagate to the postgres' `default` on the column? 
Not sure it should, just trying to figure out if it's better to generate uuid in the app code or in the DB.

Reply:
It won't - the default function runs app-side. I don't have a strong feeling on this, so I'm happy to go with whatever you think is best.

Reply:
I guess it won't work with postgres default - the value will only be available after an `insert` so no way to return it without either querying or sql `returning token`.
So I think app-side is fine.